spec/validations/translations_spec.rb

```diff
@@ -58,7 +58,7 @@
   context 'validate translation format when "package" is defined inside "app"' do
     context 'all the leaf nodes have defined "title" and "value"' do
       let(:translation_files) do
-        [double('AppFile', relative_path: 'translations/en-US.json', read: read_fixture_file('valid_en.json'))]
+        [double('AppFile', relative_path: 'translations/en-US.json', read: read_fixture_file('valid_en-US.json'))]
```

Comment:
Was there a reason you chose to create additional fixture files for these tests? For the purposes of the spec acting as documentation, in this case I would probably have just defined those _invalid_/_valid_ json structures as hashes inline with the rest of the spec